app-laptop/thinkfan/thinkfan-0.9.3.ebuild

```diff
@@ -18,17 +18,21 @@ IUSE="atasmart"
 DEPEND="atasmart? ( dev-libs/libatasmart )"
 RDEPEND="${DEPEND}"
 
-S="${WORKDIR}"
-
 src_prepare() {
-	sed -e "s:share/doc/${PN}:share/doc/${P}:" \
+	cmake-utils_src_prepare
+
+	sed -e "s:#!/sbin/runscript:#!/sbin/openrc-run:" \
+		-i rcscripts/thinkfan.gentoo
+
+	sed -e "s:share/doc/${PN}:share/doc/${PF}:" \
+	    -e "s:thinkfan.1:src/thinkfan.1:" \
 		-i CMakeLists.txt
 }
 
 src_configure() {
-	mycmakeargs+=(
+	local mycmakeargs+=(
 		"-DCMAKE_BUILD_TYPE:STRING=Debug"
```

Comment:
I general, you should avoid setting `CMAKE_BUILD_TYPE` and rather fix the build system. Not my call though.